test/runnable/traits.d

```diff
@@ -614,6 +614,21 @@ void test23()
    assert(toString23(w) == "alembicated");
 }
 
+void test24() {
+    import core.atomic;
+    import std.algorithm;
```

Comment:
I was looking for something that had some "plain" packages plus modules and "package.d", but I suppose a cleaner solution would be to make a module in test/runnable/imports. I'll fix that.